install.sh

```diff
@@ -22,12 +43,11 @@ export CCM_PATH="$(pwd)/ccm"
 export INSTALL_DIR="${HOME}/.ccm/repository/${CCM_VERSION}"
 echo ${INSTALL_DIR}
 mkdir -p ${INSTALL_DIR}
-wget ${SERVER_PACKAGE_URL} -O server.tar.gz
-tar xzf server.tar.gz -C ${INSTALL_DIR} --strip-components=1 || exit
+wget ${SERVER_PACKAGE_URL} -O ${SERVER_FILENAME}
+tar xzf ${SERVER_FILENAME} -C ${INSTALL_DIR} --strip-components=1 || exit
```

Comment:
This change was not required; however, if caching is enabled this can be moved into a sub-folder and checked to see if downloading is required.